pages/concepts/ha.mdx

```diff
@@ -0,0 +1,58 @@
+# High Availability
+
+For production deployments where the availability needs are greater than can be
+met by a single instance, ReadySet can be run in a configuration providing
+**High Availability**, or **HA**.
+
+## Running ReadySet in HA mode
+
+In order to run ReadySet in HA mode, you'll need a minimum of three ReadySet
+instances - one to run the server, and two (or more) to run the ReadySet
+adapters.
```

Comment:
nit: I think a sentence here like "The adapters will run in embedded-readers mode, meaning that the full result sets will be stored on the adapter instances, replicated across the number of adapters running" would help give context to the flag changes below.

Reply:
Done.